src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -195,8 +195,10 @@ private void buildNewQuestionForm(FeedbackSessionAttributes feedbackSession, int
                     isSelected = isGiverType || isRecipientType;
                 }
                 
-                ElementTag optionTag = createOption(participantName, option.toString(), isSelected);
-                result.add(optionTag);
+                if (option != FeedbackParticipantType.CUSTOM || isSelected) {
```

Comment:
maybe extract a boolean variable or use comment like `don't add option for custom feedback type unless it is already selected`